app/components/message-selector.tsx

```
@@ -144,60 +147,60 @@ export function MessageSelector(props: {
           type="text"
           placeholder={Locale.Select.Search}
-          className={clsx(styles["filter-item"], styles["search-bar"])}
+          className={clsx(styles['filter-item'], styles['search-bar'])}
           value={searchInput}
           onInput={(e) => {
             setSearchInput(e.currentTarget.value);
             doSearch(e.currentTarget.value);
           }}
-        ></input>
+        >
+        </input>
```

Biome (1.9.4)

[error] input is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)